src/app/pages/Details/Details.tsx

```diff
@@ -13,6 +14,10 @@ export default function Details(): JSX.Element {
 
   localStorage.setItem('CominFromDetailsPage', 'true')
 
+  function getRadius(x: number) {
+    console.log('Search Radius in meters', x)
```

Comment:
Console.log 🔍 still needed?

Reply:
for now: Yes, right now its the only thing the fuction does :)